### ISSUE TITLE
Accessibility: Missing required WAI-ARIA states or properties

### ISSUE THREAD

**Issue by eserkansozer** [corejavascript/typeahead.js#241]
Version: 1.3.1
Browser: Chrome
Plugin to run: Siteimprove

It seems there is an accessibility issue which prevents WCAG 2 A standard compatibility.

Error description from Siteimprove accessibility testing tool report based on WCAG 2:

"The WAI-ARIA element does not follow the specification in regards to which states and properties should be present for this type of element. Check the W3C specification for "Accessible Rich Internet Applications Suite" (WAI-ARIA) to see which states and properties are required for this WAI-ARIA element."

An **aria-controls** attribute should be added to the input element with value of the id of the "listbox/menu" to fix the accessibility issue.

**Comment by jlbooker** [comment]
Released in v1.3.3.
